tests/integration/targets/rds_instance_aurora/aliases

```diff
@@ -0,0 +1,6 @@
+# See: https://github.com/ansible-collections/amazon.aws/pull/1192
+disabled
+cloud/aws
+rds_instance_info
```

Comment:
`rds_instance_info` doesn't seem to me be used in this role.

Reply:
Nice catch, it was also the case of rds_instance_complex, reds_instance_modify, rds_instance_processor, rds_instance_replica, rds_instance_restore rds_instance_sgroups, rds_instance_tagging, rds_instance_upgrade